src/Illuminate/Log/Writer.php

```diff
@@ -238,7 +238,7 @@ public function useDailyFiles($path, $days = 0, $level = 'debug')
      *
      * @param  string  $name
      * @param  string  $level
-     * @return void
+     * @return \Psr\Log\LoggerInterface
```

Comment:
Actually it returns a Monolog instance, but I think hinting an interface is better here.